backend/fms_core/viewsets/library.py

```diff
@@ -29,57 +29,7 @@
 from ._constants import _library_filterset_fields
 
 class LibraryViewSet(viewsets.ModelViewSet, TemplateActionsMixin, TemplatePrefillsMixin, FetchLibraryData):
-    queryset = Sample.objects.select_related("container").filter(derived_samples__library__isnull=False).all().distinct()
-    queryset = queryset.annotate(
-        qc_flag=Case(
-            When(Q(quality_flag=False) | Q(quantity_flag=False), then=False),
-            When(Q(quality_flag=True) | Q(quantity_flag=True), then=True),
-            default=None,
-            output_field=BooleanField()
-        )
-    )
-    queryset = queryset.annotate(
-        count_derived_samples=Count("derived_samples")
-    )
-    queryset = queryset.annotate(
-        quantity_ng=F('concentration')*F('volume')
-    )
-    queryset = queryset.annotate(
-        first_volume_ratio=Subquery(
-            DerivedBySample.objects
-            .filter(sample=OuterRef("pk"))
-            .values_list("volume_ratio", flat=True)[:1]
-        )
-    )
-    queryset = queryset.annotate(
-        first_project_id=Subquery(
-            DerivedBySample.objects
-            .filter(sample=OuterRef("pk"))
-            .values_list("project_id", flat=True)[:1]
-        )
-    )
-    queryset = queryset.annotate(
-        is_pooled=Case(
-            When(Q(first_volume_ratio__lt=1) | Q(count_derived_samples__gt=1), then=True),
-            default=False,
-            output_field=BooleanField()
-        )
-    )
-    queryset = queryset.annotate(
-        sample_strandedness=Case(
-            When(Q(is_pooled__exact=True), then=None),
-            When(Q(is_pooled__exact=False), then=F('derived_samples__library__strandedness')),
-            default=None,
-            output_field=CharField())
-    )
-    queryset = queryset.annotate(
-        first_derived_sample=Subquery(
-            DerivedBySample.objects
-            .filter(sample=OuterRef("pk"))
-            .values_list("derived_sample", flat=True)[:1]
-        )
-    )
-
+    queryset = Sample.objects.all()
```

Comment:
It would make sense to at least filter queryset so that you're left with libraries.